src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -455,6 +446,30 @@ private static BidderResponse rejectBidderResponseOrProceed(
                         .with(bids));
     }
 
+    private Future<CategoryMappingResult> createCategoryMapping(AuctionContext auctionContext,
+                                                                List<BidderResponse> bidderResponses) {
+
+        if (categoryMappingService != null) {
+            return categoryMappingService.createCategoryMapping(
+                            bidderResponses,
+                            auctionContext.getBidRequest(),
+                            auctionContext.getTimeout())
+
+                    .map(categoryMappingResult -> addCategoryMappingErrors(categoryMappingResult, auctionContext));
+        }
+
+        return Future.succeededFuture(CategoryMappingResult.of(bidderResponses));
```

Comment:
what about adding interface for category mapping and replace this null check with call to NoOpCategoryMapper?